kernelci/lab/lava/lava_rest.py

```diff
@@ -38,6 +38,8 @@ class LavaRest(LavaAPI):
 
     def _get_response(self, url):
         resp = self._server.session.get(url)
+        if not resp.ok:
+            print(f'_get_response() failed, code: {resp.status_code} body: {resp.text}')
```

Comment:
We shouldn't be printing things in the library methods, instead we should be returning the string with the error or raising an exception with it.  The issue with `resp.raise_for_status()` is that it doesn't include the detailed error message from the response, just a generic one.

Reply:
Updated, can you please check if it is better now?